hudi-client/src/test/java/org/apache/hudi/table/TestHoodieMergeOnReadTable.java

```diff
@@ -90,6 +90,8 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class TestHoodieMergeOnReadTable extends HoodieClientTestHarness {
```

Comment:
@satishkotha Does this test make sense to add to hudi-hadoop-mr where the inputformats are being tested ?

Reply:
I liked hudi-client because i could validate records contents in addition to metadata.  But, thanks for the feedback, added sanity test in hadoop-mr as well.